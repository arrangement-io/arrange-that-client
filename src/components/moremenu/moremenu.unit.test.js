import React from 'react';
import Enzyme, {render} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import MoreMenu from './moremenu';

Enzyme.configure({adapter: new Adapter()});

it('moremenu loads', () => {
    const optionsList = ["option"]
    const moremenu = render(<MoreMenu options={optionsList}/>);
    expect(toJson(moremenu)).toMatchSnapshot();
});
