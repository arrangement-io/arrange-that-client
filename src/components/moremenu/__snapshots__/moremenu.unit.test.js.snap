// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`moremenu loads 1`] = `
<div>
  <button
    aria-haspopup="true"
    aria-label="More"
    class="MuiButtonBase-root-7 MuiIconButton-root-1"
    tabindex="0"
    type="button"
  >
    <span
      class="MuiIconButton-label-6"
    >
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root-10"
        focusable="false"
        role="presentation"
        viewBox="0 0 24 24"
      >
        <path
          d="M0 0h24v24H0z"
          fill="none"
        />
        <path
          d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
        />
      </svg>
    </span>
    <span
      class="MuiTouchRipple-root-19"
    />
  </button>
</div>
`;
